constraints/src/main/java/gov/nasa/jpl/aerie/constraints/model/LinearEquation.java

```diff
@@ -27,7 +27,8 @@ public LinearEquation(final Duration initialTime, final double initialValue, fin
   }
 
   public double valueAt(final Duration time) {
-    return this.initialValue + this.rate*(time.minus(this.initialTime)).ratioOver(Duration.SECOND);
+    final var change = this.rate*time.ratioOver(Duration.SECOND) - this.rate*this.initialTime.ratioOver(Duration.SECOND);
+    return this.initialValue + change;
```

Comment:
Can you explain why you made that change ?

Reply:
There is a possible long overflow in `time.minus(this.InitialTime)` that came up in testing. If `this.rate >= 1` then the overflow is unavoidable, but in this case rate was 0, so by distributing it we can avoid some overflows.